libs/cpa/common/ngx/src/lib/modules/create/components/scenarios-list/scenarios-list.component.spec.ts

```diff
@@ -1,16 +1,42 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+import { CommonModule } from '@angular/common';
+import { ActivatedRoute } from '@angular/router';
+
+import { UILayoutModule } from '@tamu-gisc/ui-kits/ngx/layout';
+import { EnvironmentModule, env } from '@tamu-gisc/common/ngx/environment';
+import { NavigationBreadcrumbModule } from '@tamu-gisc/ui-kits/ngx/navigation/breadcrumb';
 
 import { ScenariosListComponent } from './scenarios-list.component';
+import { of } from 'rxjs';
 
 describe('ScenariosListComponent', () => {
   let component: ScenariosListComponent;
   let fixture: ComponentFixture<ScenariosListComponent>;
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [ ScenariosListComponent ]
-    })
-    .compileComponents();
+      imports: [
+        CommonModule,
+        RouterTestingModule,
+        UILayoutModule,
+        NavigationBreadcrumbModule,
+        HttpClientTestingModule,
+        EnvironmentModule
+      ],
+      declarations: [ScenariosListComponent],
+      providers: [
+        {
+          provide: env,
+          useValue: { api_url: 'https://' }
+        },
+        {
+          provide: ActivatedRoute,
+          useValue: { params: of({ route: '' }) }
+        }
```

Comment:
Unnecessary provider. Only needed when `ActivateRoute` is injected into the component.